lib/utils.ts

```diff
@@ -0,0 +1,2 @@
+export { processFields, processInfo } from './definition/component-info';
```

Comment:
Not sure about these exports. They are a 'public' API so this would make sense. Maybe we should rename `index.ts` to `validate.ts` to have separate API entry points?

Reply:
This has been done in #79